moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -586,15 +595,22 @@ const ProjectFundingTable = () => {
             />
```

Comment:
just pointing out that that i was surprised to see this entire feature crammed into the `title` prop of material table 🤔

Reply:
😬